Quaver.Shared/Graphics/Graphs/DifficultySeekBar.cs

```diff
@@ -123,21 +123,15 @@ public override void Update(GameTime gameTime)
             // Handle dragging in the song
             if (IsHeld && MouseManager.CurrentState.LeftButton == ButtonState.Pressed)
             {
-                if (!Track.IsDisposed)
-                {
-                    var percentage = (MouseManager.CurrentState.Y - AbsolutePosition.Y) / AbsoluteSize.Y;
-                    var targetPos = (1 - percentage) * Track.Length;
-
-                    if ((int) targetPos != (int) Track.Time && targetPos >= 0 && targetPos <= Track.Length)
-                    {
-                        if (Math.Abs(Track.Time - targetPos) < 500)
-                            return;
+                var percentage = (MouseManager.CurrentState.Y - AbsolutePosition.Y) / AbsoluteSize.Y;
+                var targetPos = (1 - percentage) * Track.Length;
 
-                        Track.Seek(targetPos);
-                        AudioSeeked?.Invoke(this, new SeekBarAudioSeekedEventArgs());
-                    }
-                }
+                SeekToPos(targetPos);
             }
+            else if(MouseManager.CurrentState.ScrollWheelValue > MouseManager.PreviousState.ScrollWheelValue)
+                SeekToPos(Track.Time + 1000);
+            else if (MouseManager.CurrentState.ScrollWheelValue < MouseManager.PreviousState.ScrollWheelValue)
+                SeekToPos(Track.Time - 1000);
```

Comment:
You'll only want to run this if the seek bar is hovered. At its current state, it always seeks in song select when scrolling and breaks seeking entirely in the editor.